Mage.Sets/src/mage/cards/e/EnormousEnergyBlade.java

```diff
@@ -0,0 +1,61 @@
+package mage.cards.e;
+
+import mage.abilities.common.AttachedToCreatureSourceTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.common.TapEnchantedEffect;
+import mage.abilities.effects.common.continuous.BoostEquippedEffect;
+import mage.abilities.keyword.EquipAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SubType;
+
+import java.util.UUID;
+
+/**
+ *
+ * @author Addictiveme
+ */
+public final class EnormousEnergyBlade extends CardImpl {
+
+    public EnormousEnergyBlade(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ARTIFACT}, "{2}{B}");
+        
+        this.subtype.add(SubType.EQUIPMENT);
+        
+        // Equipped creature gets +4/+0
+        this.addAbility(new SimpleStaticAbility(new BoostEquippedEffect(4, 0)));
+        
+        // Whenever Enormous Energy Blade becomes attached to a creature, tap that creature.
+        this.addAbility(new AttachedToCreatureSourceTriggeredAbility(new TapAttachedEffect(), false));
+        
+        // Equip {2}
+        this.addAbility(new EquipAbility(2));
+    }
+
+    private EnormousEnergyBlade(final EnormousEnergyBlade card) {
+        super(card);
+    }
+
+    @Override
+    public EnormousEnergyBlade copy() {
+        return new EnormousEnergyBlade(this);
+    }
+}
+
+class TapAttachedEffect extends TapEnchantedEffect {
+	
+	public TapAttachedEffect() {
+		super();
+		this.staticText = "tap that creature";
```

Comment:
There's no need to make a class just to change text.  Just use `.setText()`

Reply:
Sure, but I thought it helped with clarity of the effect. The TapEnchantedEffect class performed the needed effect but it didn't seem right using it with an equipment.

No worries though, I'll change it.

Reply:
If you want to clean up that class, the best way would be to rename it to `TapAttachedEffect` and then have it take in the `AttachmentType` enum as a constructor argument.  Then use that to set the text accordingly.  See `AttacksIfAbleAttachedEffect` as an example.

For this card, it doesn't actually use the "equipped" verb so you would end up needing to set custom text regardless.

Also, when you write a custom class for a card, they're almost always named something like `EnormousEnergyBladeEffect` or something so it's clear that it's only used for that card (and not a generic class in use globally).

What you've got now looks good though so I'm happy to merge it as is.

Reply:
Nice, I'll keep what you said in mind for the future though, thanks for the feedback!